virotour/lib/src/tour/tour.dart

```diff
@@ -1,6 +1,8 @@
 // A placeholder class that represents an entity or model.
 class Tour {
-  const Tour(this.id);
+  final String tourName;
+  final String description;
+  final String id;
 
-  final int id;
+  Tour({required this.tourName, required this.description, required this.id});
```

Comment:
Should only require tour name and id. Description should be optional.

Reply:
Why would we make descriptions optional?  It would make the tours list look like something didn't load properly with the whitespace below the name.

Reply:
What do you mean by "didn't load properly"? If the tour is listed on the main screen, it's loaded.

Descriptions should be optional because some tours/items' names are self-explanatory. E.g.: Aerospace Museum Tour.

Either way works. You can also bring this up in the group meeting.

Reply:
I was talking about it giving the impression that something didn't load properly (like this image):
![image](https://user-images.githubusercontent.com/49327729/221036168-bf379db9-a4bd-4539-98f8-ba177a4f6b29.png)
To me, it looks odd having some with/some without, but totally up to getting opinions.

Reply:
I see what you mean. To me they seem fine. If you change the title of the tour to be Title Case, they'll look nicer, or make them bold or significantly bigger.

You can also bring up during our team meeting. Someone might have a better design idea. As for the code, it'll be a minor change.

Reply:
I changed the name to title case.